tests/aggregation/tests.py

```diff
@@ -988,6 +988,30 @@ def test_annotated_aggregate_over_annotated_aggregate(self):
         with six.assertRaisesRegex(self, FieldError, "Cannot compute Sum\('id__max'\): 'id__max' is an aggregate"):
             Book.objects.annotate(Max('id')).annotate(Sum('id__max'))
 
+        class MyMax(Max):
+            def as_sql(self, compiler, connection):
+                self.set_source_expressions(
+                    self.get_source_expressions()[0:1])
+                return super(MyMax, self).as_sql(compiler, connection)
+
+        with six.assertRaisesRegex(self, FieldError, "Cannot compute Max\(...'id__max'...\): 'id__max' is an aggregate"):
```

Comment:
What I was trying to convey here is that `id__max` isn't the only argument to `Max()`, but it is the argument that is wrong. When there is only one argument the error is clear:

> Cannot compute Sum('id__max'): 'id__max' is an aggregate

But when there are multiple arguments like `Sum('id__max', 'other')`, I thought that the original error message above would be a little confusing, hence the ellipsis:

> Cannot compute Sum(...'id__max'...): 'id__max' is an aggregate

But the ellipsis looks weird here, and doesn't really help to show that other arguments are involved. I'm just going to remove this special casing, and stick with the original error in all cases.

And yes, including the `id__max` part twice makes the error easier to understand. It shows that the argument to the aggregate is wrong, not the aggregate itself.